pkg/apply/event/event.go

```diff
@@ -110,6 +113,7 @@ const (
 type StatusEvent struct {
 	Type     StatusEventType
 	Resource *pollevent.ResourceStatus
+	Error    error
```

Comment:
Not sure if this is needed. The `*pollevent.ResourceStatus` already includes any errors. If we wanted to include the error directly on the `StatusEvent` instead, we should consider if we want to use a different type than `*pollevent.ResourceStatus` from the polling library.

Reply:
I wasn't aware of that. Then we don't need to include the error here.

Reply:
fixed